lib/ts/recipe/session/recipeImplementation.ts

```diff
@@ -487,7 +479,6 @@ export default function getRecipeInterface(
         ) {
             const sessionInfo = await this.getSessionInformation({
                 sessionHandle: input.sessionHandle,
```

Comment:
build needs to get tenant id as well. You can remove the optional part of userContext in types file and then compile and see all such places.

Reply:
Done